compiler+runtime/src/cpp/jank/c_api.cpp

```diff
@@ -394,7 +394,7 @@ extern "C"
   jank_object_ptr jank_list_create(uint64_t const size, ...)
   {
     /* NOLINTNEXTLINE(cppcoreguidelines-pro-type-vararg) */
-    va_list args;
+    va_list args{};
```

Comment:
Happy to merge the changes in this file, but not in `lex.cpp`. Those are out of date and not what we want.